README.md

```diff
@@ -59,19 +93,31 @@ In addition, DBSP supports **windowing operators** that group time series data
 into time windows, including various forms of tumbling and sliding windows,
 windows driven by watermarks, etc.
 
-## Theory
+## Architecture
 
-Delivering this functionality with strong performance and correctness guarantees
-requires building on a solid foundation.  The theory behind DBSP is described in
-the accompanying paper:
+The following diagram shows the architecture of the DBSP platform.  Solid
+blocks indicate components that we are currently working on; white blocks with
+dashed borders are on our TODO list.
 
-- [Budiu, McSherry, Ryzhyk, Tannen. DBSP: Automatic Incremental View Maintenance
-  for Rich Query Languages](https://arxiv.org/abs/2203.16684)
+<p align="center">
```

Comment:
the diagram is too specific about python and scala. Perhaps DDlog will come first?
I would put "other query languages"

Reply:
same for the adapters. Instead of postgres you should perhaps put ODBC and CDC